hal/src/main/java/edu/wpi/first/hal/HALUtil.java

```diff
@@ -72,6 +72,10 @@ public final class HALUtil extends JNIWrapper {
   /**
    * Reads the microsecond-resolution timer on the FPGA.
    *
+   * <p>Warning: the User Button is used to stop user programs from automatically loading if it is
+   * held for more then 5 seconds. Because of this, it's not recommended to be used by teams for any
+   * other purpose.
+   *
```

Comment:
Is this warning on the right method? (It's in the `getFPGATime()` docs, but it looks like `getFPGAButton()` 2 methods down would be more appropriate)